core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -713,8 +718,15 @@ private[spark] class BlockManager(
   /**
    * Release a lock on the given block.
    */
-  def releaseLock(blockId: BlockId): Unit = {
-    blockInfoManager.unlock(blockId)
+  def releaseLock(blockId: BlockId): Unit = releaseLock(blockId, taskAttemptId = None)
```

Comment:
I think there's only one implementation of `BlockDataManager` these days, though? Since that's an internal interface maybe we could change it there, too?